### PR TITLE
AT_01.05_006 | <Header> User should see all available "tasks" in a side panel

### PR DESCRIPTION
**Pull request by DimaCurkanu**
https://trello.com/c/Wpk94QE5/2197-rf-pom010506-header-user-should-see-all-available-tasks-in-a-side-panel

### CODE REVIEW

**Comment by DimaCurkanu** [comment]
[[RF] POM_01.05.06 | <Header> User should see all available "tasks" in a side panel](https://trello.com/c/Wpk94QE5/2197-rf-pom010506-header-user-should-see-all-available-tasks-in-a-side-panel)

File: cypress/pageObjects/UserConfigurePage.js [hunk 1]
```diff
@@ -28,5 +31,14 @@ class UserConfigurePage {
...
     }
+
+    verifyUserSidePanelItems() {
+        this.getUserSidePanelItems()
+        .then(($els) => {
+            return Cypress.$.makeArray($els).map($el => $el.innerText);
+        })
+        .should('deep.equal', userConfigurePageData.SidePanelTasks.Names)
+        return this;
+     }
```
Comment (Rom-T) [inline cypress/pageObjects/UserConfigurePage.js:42]: And if you change the method and take the assert into the test itself
```
listUserSidePanelItems() {
        return this.getUserSidePanelItems()
        .then(($els) => {
            return Cypress.$.makeArray($els).map($el => $el.innerText);
        })
     }
```

Comment (DimaCurkanu) [reply cypress/pageObjects/UserConfigurePage.js:42]: I tried many times to get the assert into the test itself, but I couldn't. The test works only in this way. I have seen that we are allowed to place assert in a method, as we did in the test 01.02.37.

Comment (DimaCurkanu) [reply cypress/pageObjects/UserConfigurePage.js]: transfer assert into test body = Done!

File: cypress/pageObjects/UserConfigurePage.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import UserProfilePage from "../pageObjects/UserProfilePage"
 import HomePage from "../pageObjects/HomePage";
+import userConfigurePageData from "../fixtures/pom_fixtures/userConfigurePage.json"
```
Comment (Rom-T) [inline cypress/pageObjects/UserConfigurePage.js:3]: Delete please this import

**Review by Rom-T — APPROVED** [review]

**Review by IrynaKolh — APPROVED** [review]

**Pull request merged**